src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -186,10 +187,14 @@ export class ExposureNotificationService {
     const runningDate = new Date();
 
     const lastCheckPeriod = periodSinceEpoch(lastFetchDate || addDays(runningDate, -14), hoursPerPeriod);
-    let runningPeriod = periodSinceEpoch(runningDate, hoursPerPeriod);
+    let runningPeriod = periodSinceEpoch(runningDate, hoursPerPeriod) - 1;
 
     while (runningPeriod > lastCheckPeriod) {
-      yield await this.backendInterface.retrieveDiagnosisKeys(runningPeriod).catch(() => null);
+      try {
+        yield await this.backendInterface.retrieveDiagnosisKeys(runningPeriod);
+        // eslint-disable-next-line no-empty
+      } catch {}
```

Comment:
Is there something better we can do here for error handling?

Reply:
Basically I just want to skip entries which generated error, but I want to keep going. 
This happens inside generator, it looks like for..in loop but the result is actually generator with `.next` and I want to not emit any errors. 
We can add logs, I guess, but in prod it won't have any meaningful effect

Reply:
Let's at least log in development so that there's some record of it happening.